cmd.sh

```diff
@@ -3,6 +3,6 @@
 if [ "$APP_ENV" == "DEVELOPMENT" ] || [ -z "$APP_ENV" ]; then
     gunicorn -w 4 -b 0.0.0.0:8081 wsgi:app --reload --worker-class gevent --timeout 600  # nosec
 else
-  pipenv run python wsgi.py
-  # gunicorn -b 0.0.0.0:8081 -w 4 wsgi:app --worker-class gevent
+  sleep 10
+  gunicorn -b 0.0.0.0:8081 -w 1 wsgi:app --worker-class gevent
```

Comment:
Is there a specific reason you are using 1 instead of 4?

Reply:
How the flask app was written it does not support multi threads after generating the route. With 4 workers we would get a mixture of 405 and 200.

Reply:
Big yikes!